common-files/utils/utils.mjs

```diff
@@ -25,15 +26,19 @@ export const isLocal = () => {
  *  HALF - ofuscate the second half of the content (e.g. 'private_key': 'HALF')
  */
 export const obfuscate = (object, obfuscationSettings) => {
+  if (!obfuscationSettings) {
+    throw new Error('Param obfuscationSettings is required!');
+  }
+
   if (
-    isDev() ||
-    !object ||
-    (typeof object !== 'object' && typeof object !== 'string') ||
-    !obfuscationSettings ||
     typeof obfuscationSettings !== 'object' ||
     Array.isArray(obfuscationSettings) ||
     obfuscationSettings.length === 0
   ) {
+    throw new Error('Invalid obfuscationSettings object!');
+  }
+
+  if (!object || (typeof object !== 'object' && typeof object !== 'string')) {
```

Comment:
:100: This new `obfuscate` version is spot on!!

Reply:
absolutely, great abstraction right here, spot on @israelboudoux

Reply:
Yes, thanks @imagobea and @signorecello, but this only abstracts obfuscation for the automatic HTTP req/res logging. I don't want to make this automatic for every log because of performance issues, since the obfuscation runs through all the properties of an object recursively, for a too deep object this can be a problem.